src/Database/StatsDAO.php

```diff
@@ -656,4 +656,47 @@ public function getSplittableItems() {
 			$statement->closeCursor();
 		}
 	}
+
+	/**
+	 * Get a list of the most common feature
+	 *
+	 * @param String $filter Feature name
+	 * @param ItemWithCode $location
+	 * @param int $limit The number of feature returned
+	 *
+	 * @return array int[] value => count, sorted by count descending
+	 */
+	public function getCommonFeature(String $filter, ?ItemWithCode $location = null, int $limit = 5) : array {
+		$locationFilter = $this->filterLocation($location);
+
+		$statement = $this->getPDO()->prepare(
+			<<<EOQ
+				SELECT ValueText AS Isa, Count(Code) AS Quantity
```

Comment:
We need to make this method more generic by:

- adding the usual `SELECT COALESCE(Value, ValueText, ValueEnum, ValueDouble) as Val` (also, Value has to be between backticks but I can't escape them in markdown asd) *if* it makes sense to aggregate on numeric values... maybe if we want to know ho many 5400 and 7200 rpm HDDs we have, and the like?
- renaming `$filter` to `$feature`, since it contains a feature name... Also remember to escape it! `$pdo->quote($feature)`
- adding a `$filter` parameter to use in the subquery
- adding a `bool $deleted = false` to filter deleted items

All these changes are very similar to parts of `getCountByFeature` so you can look at that for for code that can be reused